src/server/game/Spells/SpellEffects.cpp

```diff
@@ -4379,6 +4379,9 @@ void Spell::EffectEnchantHeldItem(SpellEffIndex effIndex)
         if (!duration)
             duration = 10;                                  //10 seconds for enchants which don't have listed duration
```

Comment:
should multiply this value by IN_MILLISECONDS (since SetEnchantment expects value in ms and it was removed from line 4397 as it didnt make sense to mutliply a value already in ms (from spell duration) again)